### PR TITLE
Spike dw graph ql r spec

### LINKED ISSUE #42: RSpec test query for one Organization endpoint

### PR DESCRIPTION
**Pull request by DavisWeimer**
## Explain changes:
Add's the `rspec-graphql_matchers` gem plus the beginnings of the Get One Organization interface-level endpoint test

### What does this PR do?
- [ ] Feature
- [ ] Bug fix
- [ ] Refactor
- [ ] Styling
- [x] Other:

### Necessary checkmarks:
- [ ] All Tests are Passing
- [ ] The code will run locally

### Testing Changes
- [ ] No Tests have been changed/added
- [x] Some Tests have been changed/added
- [ ] All of the Tests have been changed(Please explain)

## Issues this closes 
#42

### CODE REVIEW

File: spec/graphql/queries/query_type_spec.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      expect(response["shareAddress"]).to be_in([true, false])
+
+      expect(response).to have_key("sharePhone")
+      expect(response["sharePhone"]).to be_in([true, false])
+
+      expect(response).to have_key("shareEmail")
+      expect(response["shareEmail"]).to be_in([true, false])
+
+      expect(response).to have_key("aidRequests")
+      response
```
Comment (ethanrossblack) [inline spec/graphql/queries/query_type_spec.rb:61]: unfinished tests?

File: Gemfile [hunk 2]
```diff
@@ -59,3 +59,6 @@ group :development do
   gem "graphiql-rails"
 end
 
+group :test do
+  gem 'rspec-graphql_matchers'
```
Comment (ethanrossblack) [inline Gemfile:63]: Good find!

File: spec/graphql/queries/query_type_spec.rb [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    end
+  end
+
+  def get_one_query
```
Comment (ethanrossblack) [inline spec/graphql/queries/query_type_spec.rb:65]: rename to `get_one_organization_query`

File: spec/graphql/queries/query_type_spec.rb (hunk 1 above)
Comment (DavisWeimer) [reply spec/graphql/queries/query_type_spec.rb]: Oop! Updated

File: spec/graphql/queries/query_type_spec.rb (hunk 3 above)
Comment (DavisWeimer) [reply spec/graphql/queries/query_type_spec.rb]: Added name change to method

**Review by ethanrossblack — APPROVED** [review]
Thanks for making those few changes! Good start to query testing.

**Pull request merged**